synapse/tests/test_lib_stormtypes.py

```diff
@@ -1831,8 +1823,8 @@ async def test_storm_lib_layer(self):
 
                 layr = core.getLayer(uplayr)
 
-                evnt = await layr.waitUpstreamOffs(layriden, offs)
-                await asyncio.wait_for(evnt.wait(), timeout=2.0)
+                #evnt = await layr.waitUpstreamOffs(layriden, offs)
```

Comment:
should have some sort of alternative here; either breaking it out into a separate test or a different way to validate the upstream behaviors.